### PR TITLE
just added more functions and variables

### PR DESCRIPTION
**Pull request by XD3V**
I added more a function for validating no info. I also added new variables for showModal. As well as adding a current date.

### CODE REVIEW

**Review by SirHexxus — CHANGES_REQUESTED** [review]
Line 106-107: Should be `||` not `&`

**Review by XD3V — COMMENTED** [review]
I added the new library for us as well as make change to the code from changin "&" to "||"

**Review by XD3V — COMMENTED** [review]
in this push I am just showing how the RegExp is working to validate the users response. Also, i found and fixed that typo on line 121 with the 2 "||" so you will not have to worry about that on the next push.

**Review by XD3V — COMMENTED** [review]
Also, I moved the pseudo code for the event cards in the submit button so, it can all be done in one call

File: app.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    userCity.val().trim();
+
+    userState.val();
+
+    userDate.val();
+
+    pickedTime.val();
+
+    //
+   if( (userDate == "" ) & (userState == "") & (pickedTime == "") & (userCity == "")){
```
Comment (SirHexxus) [inline app.js]: status = true

if ( userDate == "" ) {
  // Please Enter Date
  status = false
}

if ( userState == "" ) {
  // Please Enter State
  status = false
}

if ( pickedTime == "" ) {
  // Please Choose Time
  status = false
}

if ( userCity == "" ) {
  // Please Enter City
  status = false
}

**Review by SirHexxus — CHANGES_REQUESTED** [review]

File: app.js [hunk 2]
```diff
@@ -102,7 +104,7 @@ const submitBtn = function submit() {
 
     pickedTime.val();
 
-    //
+    // check to make sure all requirements are met so they can go to the next page
    if( (userDate == "" ) || (userState == "") || (pickedTime == "") || (userCity == "")){
```
Comment (SirHexxus) [inline app.js:108]: Change this to individual conditions, with `status = true;` by default, and if any condition is true, it sets `status = false;`

File: app.js [hunk 3]
```diff
@@ -148,20 +156,9 @@ const clearBtn = function clear() {
...
-// Card-Event-2
-
-// Card-Event-3
 
 /* Second page */
 
-// check to make sure all requirements are met so they can go to the next page
+
 
 <script src="VerbalExpressions.js"></script>
```
Comment (SirHexxus) [inline app.js:164]: Remove, and add to Index.html

**Review by SirHexxus — APPROVED** [review]

**Pull request merged**